buildkitd/entrypoint.sh

```diff
@@ -233,30 +233,21 @@ echo "======== End buildkitd config =========="
 
 echo "Detected container architecture is $(uname -m)"
 
-# start shell repeater server
-if [ "$DISABLE_SHELL_REPEATER" = "true" ]; then
-    echo shellrepeater is disabled
-    shellrepeaterpid=0
-else
-    echo starting shellrepeater
-    shellrepeater &
-    shellrepeaterpid=$!
-fi
-
 "$@" &
 execpid=$!
 
 # quit if either buildkit or shellrepeater die
 set +x
 while true
 do
-    if [ "$shellrepeaterpid" != "0" ]; then
-        if ! kill -0 "$shellrepeaterpid" >/dev/null 2>&1; then
-            wait "$shellrepeaterpid"
-            echo "Error: shellrepeater process has exited with code $?"
-            exit 20
-        fi
-    fi
+    # shellrepeater is no longer needed -- but perhaps this pattern is useful?
+    #if [ "$shellrepeaterpid" != "0" ]; then
+    #    if ! kill -0 "$shellrepeaterpid" >/dev/null 2>&1; then
+    #        wait "$shellrepeaterpid"
+    #        echo "Error: shellrepeater process has exited with code $?"
+    #        exit 20
+    #    fi
+    #fi
```

Comment:
I would just remove this altogether. We can always pull it back from git history in the future.